### ISSUE TITLE
Vue 2.6.11 - Unknown custom element: <Particles>

### ISSUE THREAD

**Issue by tonystrawberry** [tsparticles/tsparticles#1221]
**Which version are you using?**
Ex. 2.1.11

**Is the latest version affected?**

- [X] Yes
- [ ] No

**Which library are you using?**

- [ ] Vanilla JS (`tsparticles`)
- [ ] React (`react-tsparticles`)
- [X] Vue.js 2.x (`particles.vue`)
- [ ] Vue.js 3.x (`particles.vue3`)
- [ ] Angular (`ng-particles`)
- [ ] Svelte (`svelte-particles`)
- [ ] Preact (`preact-particles`)
- [ ] Inferno (`inferno-particles`)
- [ ] jQuery (`jquery-particles`)

**Describe the bug**
Hello! I followed the instructions for installing tsparticles on Vue 2.X here: https://github.com/matteobruni/tsparticles/blob/master/components/vue/README.md

But after installing it, I got the following error here. 

```
Unknown custom element: <Particles> - did you register the component correctly? For recursive components, make sure to provide the "name" option.
```

Have you ever met this problem before? Thank you!

**Desktop (please complete the following information):**
 - OS: [MacOS]
 - Browser [Chrome]
 - Version [87.0.4280.88]
 
Thank you! 🙇‍♂️

**Comment by matteobruni** [comment]
Hi @tonystrawberry,

Can you share some code, please? So I can replicate the issue and identify the problem.

Thanks

**Comment by mtpiercey** [comment]
This is more of a workaround then a fix, but I thought of it the second I saw this issue's name.

I had a very similar issue with importing Ionicons in Nuxt (using Vue 2.6.x), where Vue wasn't recognizing the `ion-icon` component. I got around the unknown custom element problem with this bit in my `nuxt.config.js`. Not sure if it's relevant, but again it worked for me in a very similar scenario, so hope it helps:

```js
  vue: {
    config: {
      ignoredElements: [
        'ion-icon'
      ]
    }
  }

```
